docs/addresses.rst

```diff
@@ -22,6 +23,43 @@ any other financial service.
 
         To install the extension, run ``pip install pyota[ccurl]``.
 
+Algorithm
+---------
+
+.. figure:: images/address_gen.svg
+   :scale: 100 %
+   :alt: Process of address generation in IOTA.
+
+   Deriving addresses from a seed.
+
+The following process takes place when you generate addresses in IOTA:
+
+1. First, a sub-seed is derived from your seed by adding ``index`` to it,
+   and hashing it once with the `Kerl <https://github.com/iotaledger/kerl>`__
```

Comment:
```suggestion
   and hashing it once with the `Kerl <https://github.com/iotaledger/kerl>`_
```

❔ Also, wondering if it would be more consistent to make these named hyperlink targets (i.e., add them to the bottom of the document like `.. _Kerl: https://github.com/iotaledger/kerl`)?

Reply:
yes, agreed.